### ISSUE TITLE
Add capability to render markdown stored in DB

### ISSUE THREAD

**Issue by ps-89** [game1n/t8-cms#20]
- Admin would create and store the blog in form of markdown, the markdown would be stored in some BaaS like superbase(this will be handled separately)
- In this task, we want to add the capability to be able to render that markdown coming from the DB.

Ref:  https://github.com/remarkjs/react-markdown

**Comment by abhiandthetruth** [comment]
@ps-89 facing difficulty while connecting to supabase, mostly because tables do not exist. Are we supposed to construct the tables manually in supabase?
If yes, should we not have migrations etc to have this automatically constructed?

**Comment by ps-89** [comment]
There's missing doc around the DB schema, the migrations should be there ideally. What I'm thinking of is moving away from supabase or using some abstraction to hide supabase's integration underneath which could be swapped with some other backend service(cue for an interesting task to take up if someone's interested).

Will update the schema and instructions to run the project with proper env variables to point to one's own supabase URL in sometime.

**Comment by abhiandthetruth** [comment]
Yeah that is a good thought in my opinion. Making a codebase database agnostic is best. We can limit the db interaction to a specific layer in the code. Kind of a wrapper on the db calls. That would be a major redesign tho. 
For tracking purposes you can create a separate issue for this and also for the docs too.

**Comment by ps-89** [comment]
@abhiandthetruth You can refer below screenshot for setting up your own schema on a personal supabase account, I'll setup the SQL DDL file later. Will create the issues too in some time. 

<img width="649" alt="Screenshot 2022-10-10 at 9 17 29 AM" src="https://user-images.githubusercontent.com/91064940/194796781-4543b550-1f75-4586-ab5d-1b52e640cdb5.png">

**Comment by abhiandthetruth** [comment]
Yup will try the above queries. Will raise a PR with the docs and migration

**Comment by ps-89** [comment]
@abhiandthetruth Do let me know if you're facing any issues or if more context is needed.

**Comment by abhiandthetruth** [comment]
@ps-89 will be able to start this on the weekend. Short on time rn

**Comment by ps-89** [comment]
DB setup and instructions are done. The DDL file is present at `./resources/postgres/t8_ddl.sql`.
